assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -347,6 +347,10 @@ che.infra.kubernetes.workspace_start_timeout_min=8
 # Defines the timeout in minutes that limits the period for which Kubernetes Ingress become ready
 che.infra.kubernetes.ingress_start_timeout_min=5
 
+# If during workspace startup an unrecoverable event defined in the property occurs,
+# terminate workspace immediately instead of waiting until timeout
+che.infra.kubernetes.workspace_unrecoverable_events=Failed Mount,Failed Scheduling,Failed to pull image
```

Comment:
For users it may be interesting to have a list (or better a link) of valid kubernetes events.

Reply:
@l0rd could you please clarify a bit ? is it about changing the property format

Reply:
I think Mario means a link to docs where all the event types are listed. 
And :+1:  to add it here.

Reply:
If I am a Che user and I am editing che.properties I don't know what are the valid values for this new property. You may want to say in the comment that this is a comma separated list of kubernetes events. And it would be great if we could add a link to a list of valid kubernetes events (but I am not sure we can easily find one so don't lose too much time on this)